### PR TITLE
fix: Bump minSdk version to 26

### PR DESCRIPTION
**Pull request by felipecsl**
[Jackson v2.13](https://github.com/FasterXML/jackson/wiki/Jackson-Releases) [dropped](https://github.com/FasterXML/jackson-module-kotlin/issues/716#issuecomment-1765071697) support for Android version below 26, so this was just instacrashing on Android 24 and 25 with:
```
 Process: cloud.eppo.androidexample, PID: 4545
    java.lang.NoClassDefFoundError: Failed resolution of: Ljava/lang/BootstrapMethodError;
        at com.fasterxml.jackson.databind.util.ExceptionUtil.isFatal(ExceptionUtil.java:51)
        at com.fasterxml.jackson.databind.util.ExceptionUtil.rethrowIfFatal(ExceptionUtil.java:31)
        at com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector.<clinit>(JacksonAnnotationIntrospector.java:70)
        at com.fasterxml.jackson.databind.ObjectMapper.<clinit>(ObjectMapper.java:402)
        at cloud.eppo.BaseEppoClient.<init>(BaseEppoClient.java:25)
        at cloud.eppo.android.EppoClient.<init>(EppoClient.java:42)
        at cloud.eppo.android.EppoClient.<init>(EppoClient.java:24)
        at cloud.eppo.android.EppoClient$Builder.buildAndInitAsync(EppoClient.java:210)
        at cloud.eppo.androidexample.SecondActivity.onCreate(SecondActivity.java:53)
        at android.app.Activity.performCreate(Activity.java:6662)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2599)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6077)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866)
```

Plus, with minSdk 26 we can use `java.util.Base64`, which was added on https://github.com/Eppo-exp/sdk-common-jdk/pull/60

### CODE REVIEW

**Comment by aarsilv** [comment]
Looks good, will approve once the common update is merged and tests are passing!

File: example/src/main/res/layout/activity_main.xml
Comment (felipecsl) [inline example/src/main/res/layout/activity_main.xml:37]: this fixes a crash with the example app on android 26

File: example/src/main/java/cloud/eppo/androidexample/MainActivity.java
Comment (felipecsl) [inline example/src/main/java/cloud/eppo/androidexample/MainActivity.java:34]: default offline mode to false

File: eppo/src/main/java/cloud/eppo/android/util/Utils.java
Comment (felipecsl) [inline eppo/src/main/java/cloud/eppo/android/util/Utils.java:76]: we were including commons-io literally just for this one method, so I removed the dep and included it here

**Review by aarsilv — APPROVED** [review]
🎉

**Pull request merged**
